controllers/admin/AdminModulesController.php

```diff
@@ -1415,7 +1415,13 @@ public function initModal()
         $translateLinks = [];
 
         if (Tools::getIsset('configure')) {
+            /** @var Module|false $module */
             $module = Module::getInstanceByName(Tools::getValue('configure'));
+
+            if (false === $module) {
```

Comment:
maybe `if (!$module instanceof Module)`  is more generic but not that important..:)